core/src/test/groovy/com/novoda/gradle/release/TestHelper.groovy

```diff
@@ -0,0 +1,33 @@
+package com.novoda.gradle.release
+
+import org.gradle.api.Project
+import org.gradle.testfixtures.ProjectBuilder
+
+final class TestHelper {
```

Comment:
I prefer to pull helper functions out of the test classes, but if you'd rather include it directly in the tests, that's fine